recipients_app/lib/view/pages/payments_page.dart

```diff
@@ -154,10 +157,17 @@ class _PaymentsPageState extends State<PaymentsPage> {
   }
 
   String _calculateFuturePayments(List<MappedPayment> mappedPayments) {
```

Comment:
@novas1r1  
Should be the methods "_calculatePastPayments" and "_calculateFuturePayments" really in the payment page? This logic is business logic and should be placed somewhere else. But for now I have no idea where to place it exactly. What do you mean?